<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Baradé's vJass IDE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="LineNumbers" name="lineNumbersWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="textEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Syntax Errors</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QListWidget" name="outputListWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="outliner">
       <attribute name="title">
        <string>Outliner</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QListWidget" name="outlinerListWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionClose"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAuto_Completion">
    <property name="title">
     <string>Auto Completion</string>
    </property>
    <addaction name="actionComplete"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionBaradesVJassIDE"/>
   </widget>
   <widget class="QMenu" name="menuAutomaticAnalysis">
    <property name="title">
     <string>Automatic Analysis</string>
    </property>
    <addaction name="actionEnableSyntaxHighlighting"/>
    <addaction name="actionEnableSyntaxCheck"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionLineNumbers"/>
    <addaction name="actionShowWhiteSpaces"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuOnline">
    <property name="title">
     <string>Online</string>
    </property>
    <addaction name="actionCodeOnHive"/>
    <addaction name="actionJASS_Manual"/>
    <addaction name="actionJassHelper"/>
    <addaction name="actionpjass"/>
   </widget>
   <widget class="QMenu" name="menuStandard_Scripts">
    <property name="title">
     <string>Scripts</string>
    </property>
    <addaction name="actionCommonj"/>
    <addaction name="actionCommonAi"/>
    <addaction name="actionBlizzardj"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuStandard_Scripts"/>
   <addaction name="menuView"/>
   <addaction name="menuAuto_Completion"/>
   <addaction name="menuAutomaticAnalysis"/>
   <addaction name="menuSettings"/>
   <addaction name="menuOnline"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionComplete">
   <property name="text">
    <string>Complete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionBaradesVJassIDE">
   <property name="text">
    <string>Baradé's vJass IDE</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEnableSyntaxHighlighting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Syntax Highlighting</string>
   </property>
  </action>
  <action name="actionEnableSyntaxCheck">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Syntax Check</string>
   </property>
  </action>
  <action name="actionLineNumbers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Line Numbers</string>
   </property>
  </action>
  <action name="actionShowWhiteSpaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show White Spaces</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionCodeOnHive">
   <property name="text">
    <string>Code on Hive</string>
   </property>
  </action>
  <action name="actionJASS_Manual">
   <property name="text">
    <string>JASS Manual</string>
   </property>
  </action>
  <action name="actionJassHelper">
   <property name="text">
    <string>JassHelper</string>
   </property>
  </action>
  <action name="actionpjass">
   <property name="text">
    <string>pjass</string>
   </property>
  </action>
  <action name="actionCommonj">
   <property name="text">
    <string>common.j</string>
   </property>
  </action>
  <action name="actionCommonAi">
   <property name="text">
    <string>common.ai</string>
   </property>
  </action>
  <action name="actionBlizzardj">
   <property name="text">
    <string>Blizzard.j</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineNumbers</class>
   <extends>QWidget</extends>
   <header location="global">linenumbers.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
